diff --git a/node_modules/@apidevtools/json-schema-ref-parser/lib/pointer.js b/node_modules/@apidevtools/json-schema-ref-parser/lib/pointer.js
index e637105..15a752e 100644
--- a/node_modules/@apidevtools/json-schema-ref-parser/lib/pointer.js
+++ b/node_modules/@apidevtools/json-schema-ref-parser/lib/pointer.js
@@ -91,7 +91,12 @@ Pointer.prototype.resolve = function (obj, options, pathFromRoot) {
     let token = tokens[i];
     if (this.value[token] === undefined || this.value[token] === null) {
       this.value = null;
-      throw new MissingPointerError(token, this.originalPath);
+
+      // hack 修改抛错行为，防止退出
+      console.warn(
+          `\nError resolving $ref pointer "${this.originalPath}". \nToken "${token}" does not exist.`
+      );
+      // throw new MissingPointerError(token, this.originalPath);
     }
     else {
       this.value = this.value[token];
